index.js

```diff
@@ -5,18 +5,18 @@ document.addEventListener('DOMContentLoaded', () => {
 
     // Afegeix un listener d'esdeveniments per a la presentació del formulari
     form.addEventListener('submit', (event) => {
-        event.preventDefault(); // Prevenir el comportament predeterminat de presentació del formulari
+        event.preventDefault(); 
 
         // Recupera les dades del formulari
-        const title = sanitizeInput(form.noteTitle.value);
-        const content = sanitizeInput(form.noteContent.value);
+        const title = form.noteTitle.value;
```

Comment:
It was suggested to use `event` to extract target values.

- [ ] Implement `event.target`.